File: python/types/strings/strings.py
#!/usr/bin/python3

s = "hello"
print(f"string \"hello\": {s} ({type(s)})")

s = 'hello'
print(f"string 'hello': {s} ({type(s)})")

print(f"length: {len(s)}\n")

cite = "Stat rosa pristina nomine, nomina nuda tenemus."
print(f"cite: {cite}")
print(f"words quantity: {len(cite.split())}\n")

text = """
  Videmus nunc
  per speculum
  et in aenigmate.
""";

print(f"text:{text}");

s1 = "abc"
s2 = "def"
s3 = s1 + s2
print(f"{s1} + {s2} = {s3}\n")

File: javascript/types/strings/strings.js
#!/usr/bin/env node

let s = "hello";
console.log(`string "hello": ${s} (${typeof(s)})`);

s = 'hello';
console.log(`string 'hello': ${s} (${typeof(s)})`);

console.log(`length: ${s.length}\n`);

let cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
console.log(`cite: ${cite}`);
console.log(`words quantity: ${cite.split(" ").length}\n`);

let text = `
  Videmus nunc
  per speculum
  et in aenigmate.
`;

console.log(`text:${text}`);

let s1 = "abc";
let s2 = "def";
let s3 = s1 + s2;
console.log(`${s1} + ${s2} = ${s3}\n`);
